### PR TITLE
Rise: Change the name of the mandatory field

### PR DESCRIPTION
**Pull request by zkosanovic**
We want to receive the publisher ID in the field `org` instead of the field `publisher_id`.

### CODE REVIEW

File: static/bidder-params/rise.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
   "properties": {
-    "publisher_id": {
+    "org": {
       "type": "string",
-      "description": "The publisher ID"
+      "description": "The organization ID"
     }
   },
-  "required": ["publisher_id"]
+  "required": ["org"]
```
Comment (gargcreation1992) [inline static/bidder-params/rise.json:12]: Please give publishers enough transition period to change publisher_id to org. In the meantime, you could support both. 
```
"oneOf": [
    {
      "required": [
        "publisher_id"
      ]
    },
    {
      "required": [
        "org"
      ]
    }
  ]
  ```

**Comment by gargcreation1992** [comment]
@zkosanovic Requesting you to please address the open comment.

**Comment by zkosanovic** [comment]
@gargcreation1992 Thanks for the review and a good suggestion! I have made the changes to support both fields temporarily.

File: adapters/rise/risetest/exemplary/banner-and-video-site.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
         },
         "ext": {
           "bidder": {
-            "org": "72721",
+            "publisher_id": "72721",
```
Comment (gargcreation1992) [inline adapters/rise/risetest/exemplary/banner-and-video-site.json]: Requesting you to please add at least one test case where org is not empty. Your code assumes that if both org and publisherID is provided, publisherID takes the precedence. You might wanna consider reversing your if condition; since org should take the precedence but it is for you to decide.
I would like to make a request to please [update the docs](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/rise.md) with above information(which one should take the precedence) and link the PR here.

**Review by gargcreation1992 — COMMENTED** [review]
Requesting to add a test case and update the docs.

File: adapters/rise/risetest/exemplary/banner-and-video-site.json (hunk 2 above)
Comment (zkosanovic) [reply adapters/rise/risetest/exemplary/banner-and-video-site.json]: The code takes `org` first if present. I have added a test case to show that.

For the docs, it is okay for us that they stay as they are. As you can see, the docs already mention the `org` field we are trying to transition to here.

Comment (gargcreation1992) [reply adapters/rise/risetest/exemplary/banner-and-video-site.json]: @zkosanovic thanks for being patient here. 
If you add both org and publisherID, publisherId takes the precedence. You can also add a test case to validate this. For example - 
```
"ext": {
          "bidder": {
            "publisher_id": "1",
            "org": "2",
            "path": "mvo",
            "zone": "1r"
          }
          ```
In above case, both org and publisher_id is provided but value of publisher_id will take precedence. 

For the docs, I am ok if you would not like to mention about the publisher_id.

Comment (gargcreation1992) [reply adapters/rise/risetest/exemplary/banner-and-video-site.json]: Cool! Thanks for that test case. g2g.

**Review by gargcreation1992 — DISMISSED** [review]

File: static/bidder-params/rise.json [hunk 3]
```diff
@@ -4,10 +4,13 @@
...
       "type": "string",
-      "description": "The publisher ID"
+      "description": "The organization ID"
     }
   },
-  "required": ["publisher_id"]
+  "oneOf": [
+    { "required": ["publisher_id"] },
+    { "required": ["org"] }
+  ]
```
Comment (onkarvhanumante) [inline static/bidder-params/rise.json:15]: The `publisher_id` is  mentioned as `oneOf` required. So should be mentioned  under `properties` with deprecated prefix in description. Refer following example.

https://github.com/prebid/prebid-server/blob/225e31e9afea25f8b844adcd3d274e8db3f7135e/static/bidder-params/appnexus.json#L12-L15

File: adapters/rise/rise.go [hunk 4]
```diff
@@ -43,7 +44,7 @@ func (a *adapter) MakeRequests(openRTBRequest *openrtb2.BidRequest, _ *adapters.
 
 	return append(requestsToBidder, &adapters.RequestData{
 		Method:  http.MethodPost,
-		Uri:     a.endpointURL + "?publisher_id=" + publisherID,
+		Uri:     a.endpointURL + "?publisher_id=" + org,
```
Comment (onkarvhanumante) [inline adapters/rise/rise.go:47]: Note that you are adding `org` as `publisher_id`.  Assuming this is been handled at bidder server.

File: static/bidder-params/rise.json (hunk 3 above)
Comment (zkosanovic) [reply static/bidder-params/rise.json]: @onkarvhanumante Sure, done

File: adapters/rise/rise.go (hunk 4 above)
Comment (zkosanovic) [reply adapters/rise/rise.go:47]: @onkarvhanumante Yes, it is 👍🏻

**Review by onkarvhanumante — APPROVED** [review]

**Review by onkarvhanumante — COMMENTED** [review]
@zkosanovic  tests are failing with following error.

```
--- FAIL: TestValidParams (0.02s)
    params_test.go:18: Failed to fetch the json-schemas. Failed to load json schema at /home/runner/work/prebid-server/prebid-server/static/bidder-params/rise.json: invalid character ']' looking for beginning of value
--- FAIL: TestInvalidParams (0.02s)
    params_test.go:32: Failed to fetch the json-schemas. Failed to load json schema at /home/runner/work/prebid-server/prebid-server/static/bidder-params/rise.json: invalid character ']' looking for beginning of value
```

**Review by gargcreation1992 — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]
